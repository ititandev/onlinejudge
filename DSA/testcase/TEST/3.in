10
54.35 93.7 64.07 37.9 -93.63 -6.34 25.13 70.6 -69.19 -90.1
