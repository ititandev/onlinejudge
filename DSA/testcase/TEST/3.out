  -93.63   -90.1  -69.19   -6.34   25.13
    37.9   54.35   64.07    70.6    93.7
